### PR TITLE
{bio}[intel/2016a] BCFtools 1.3 + HTSlib 1.3 (REVIEW)

### PR DESCRIPTION
**Pull request by boegel**

### CODE REVIEW

File: easybuild/easyconfigs/b/BCFtools/BCFtools-1.3-intel-2016a.eb [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+parallel = 1
+
+skipsteps = ['configure']
+
+buildopts = 'CC="$CC" CFLAGS="$CFLAGS" USE_GSL=1'
+installopts = ' prefix=%(installdir)s'
+
+postinstallcmds = [
+    'mkdir -p %(installdir)s/lib/plugins',
+    'cp plugins/*.so %(installdir)s/lib/plugins/.',
```
Comment (wpoely86) [inline easybuild/easyconfigs/b/BCFtools/BCFtools-1.3-intel-2016a.eb]: `cp -a`?

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2059.delcatty.os - Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/4d27d451f922986057f5 for a full test report.

File: easybuild/easyconfigs/h/HTSlib/HTSlib-1.3-intel-2016a.eb [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+# Swiss Institute of Bioinformatics
+# Biozentrum - University of Basel
+
+easyblock = 'ConfigureMake'
+
+name = 'HTSlib'
+version = '1.3'
+
+homepage = "http://www.htslib.org/"
+description = """ A C library for reading/writing high-throughput sequencing data.
```
Comment (wpoely86) [inline easybuild/easyconfigs/h/HTSlib/HTSlib-1.3-intel-2016a.eb]: leading and trailing whitespace,

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in this PR)
node2059.delcatty.os - Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/856aa3f0aa5b323dc4ed for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in this PR)
node2633.swalot.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/73ee34a3320689f99c27 for a full test report.

**Comment by boegel** [comment]
going in, thanks for the review @wpoely86!

**Pull request merged**